java/src/org/openqa/selenium/html5/Storage.java

```diff
@@ -0,0 +1,38 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.openqa.selenium.html5;
+
+import java.util.Set;
+
+/**
+ * Represents common operations available for all web storage types (session or local).
+ */
+public interface Storage {
```

Comment:
I am just wondering if we should call this `WebStorage` to avoid ambiguity. What do you think?

Reply:
as usual naming is one of two the most complex problems in programming :)

there is already `WebStorage` interface: https://github.com/SeleniumHQ/selenium/blob/trunk/java/src/org/openqa/selenium/html5/WebStorage.java (and its name perfectly matches specification)

various sources provides different names for `localStorage` and `sessionStorage`:
- `mechanism`
- storage `object`
- simply `storage`
- `storage area`

I'm open for any suggestion here

Reply:
Fair enough, let's leave it like that for now. Thank you.